Source/CesiumRuntime/CesiumRuntime.Build.cs

```diff
@@ -211,7 +211,12 @@ public CesiumRuntime(ReadOnlyTargetRules Target) : base(Target)
 
         PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
         PrivatePCHHeaderFile = "Private/PCH.h";
+
+#if UE_5_4_OR_LATER
+        CppStandard = CppStandardVersion.Cpp20;
```

Comment:
Just checking, is C++ 20 now the min version in 5.4?

Reply:
Good question. The plugin didn't build right away with 17, so I bumped it to 20.

The [Unreal coding standard](https://dev.epicgames.com/documentation/en-us/unreal-engine/epic-cplusplus-coding-standard-for-unreal-engine) says 17 is the minimum, but wouldn't be surprised if it was out of date.
![image](https://github.com/CesiumGS/cesium-unreal/assets/130494071/93879843-a6db-4d6a-bc60-cc4f18891316)


We could invest some time into figuring out how to get 5.4 to compile using C++17... or just bump it, which I did instead.

Reply:
Hmm, do you remember any of the errors that you saw with C++17? That will probably affect whether it's realistic to try and fix it in this timeframe.

@kring do you have any strong feelings about compiling with C++20? Just wanted to check if this had any consequences / implications we're not considering.

Reply:
Easy enough to check, just set it back and build again with 5.4.

These are the first errors you would see... 

```
2>------ Building 15 action(s) started ------
2>[1/15] Compile [x64] PCH.CesiumRuntime.cpp
2>[2/15] Compile [x64] Module.CesiumRuntime.2.cpp
2>[3/15] Compile [x64] Module.CesiumRuntime.1.cpp
2>C:\Program Files\Epic Games\UE_5.4\Engine\Source\Runtime\Engine\Public\NaniteSceneProxy.h(513): error C7582: 'bHasRayTracingInstances': default member initializers for bit-fields requires at least '/std:c++20'
2>C:\Program Files\Epic Games\UE_5.4\Engine\Source\Runtime\Engine\Public\NaniteSceneProxy.h(514): error C7582: 'bNeedsDynamicRayTracingGeometries': default member initializers for bit-fields requires at least '/std:c++20'
2>C:\Program Files\Epic Games\UE_5.4\Engine\Source\Runtime\Renderer\Private\ScenePrivate.h(2938): error C7589: defaulting comparison operators requires at least '/std:c++20'
```

Reply:
Oddly enough, those are the only errors I see when compiling with C++17. 
I just see them repeated over and over again.

Reply:
Ah well. If they require C++ 20, then that's what we need to compile with too :')

Reply:
Total guess, but it might have slipped through the cracks for the UE 5.4 release. 
Feels like so little code that requires C++20.